Cargo.lock

```diff
@@ -250,7 +250,7 @@ checksum = "8c3c1a368f70d6cf7302d78f8f7093da241fb8e8807c05cc9e51a125895a6d5b"
 
 [[package]]
 name = "benchmark-example-node"
-version = "0.2.0"
+version = "0.2.0-rc-10"
```

Comment:
These should match the versions specified in the Cargo.toml. I think you accidentally commited your lockfile after doing the test release?

Reply:
Indeed. Good catch!